sdk/include/wdf/kmdf/1.9/wdf.h

```diff
@@ -0,0 +1,45 @@
+/*
+ * wdf.h
+ *
+ * Windows Driver Framework - Primary Header File
+ *
+ * This file is part of the ReactOS wdf package.
+ *
+ * Contributors:
+ *   Created by Benjamin Aerni <admin@bennottelling.com>
+ *
+ * Intended Usecase:
+ *   Kernel mode drivers
+ *
+ * THIS SOFTWARE IS NOT COPYRIGHTED
+ *
+ * This source code is offered for use in the public domain. You may
+ * use, modify or distribute it freely.
+ *
+ * This code is distributed in the hope that it will be useful but
+ * WITHOUT ANY WARRANTY. ALL WARRANTIES, EXPRESS OR IMPLIED ARE HEREBY
+ * DISCLAIMED. This includes but is not limited to warranties of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ *
+ */
+#ifndef _WDF_H_
+#define _WDF_H_
+
+
+typedef VOID (*WDFFUNC) (VOID);
+extern WDFFUNC WdfFunctions[];
+
+#ifndef _drv_dispatchType
```

Comment:
I'm guessing it's supposed to be `__drv_dispatchType` with two underscores

Reply:
Also, what source are you basing this on? My version of the MS header checks `_Dispatch_type_` here.

Reply:
The DDK version 7